src/seedu/addressbook/data/person/Name.java

```diff
@@ -40,7 +40,7 @@ public static boolean isValidName(String test) {
      * Retrieves a listing of every word in the name, in order.
      */
     public List<String> getWordsInName() {
-        return Arrays.asList(fullName.split("\\s+"));
+        return Arrays.asList(fullName.toLowerCase().split("\\s+"));
```

Comment:
We try not to touch unrelated classes to minimise the effect our change has on the whole code base. Since we're making `find` case-insensitive, `FindCommand#getPersonsWithNameContainingAnyKeyword()` will be a better place to add this change.